config.go

```diff
@@ -20,9 +20,8 @@ func initializeConfiguration() {
 	viper.AddConfigPath("$HOME/.rv")
 	var configuration Configuration
 
-	if err := viper.ReadInConfig(); err != nil {
-		log.Fatalf("Error reading config file, %s", err)
-	}
+	_ = viper.ReadInConfig()
```

Comment:
Should we just bail early on an error here?  What happens with the lines below?

Reply:
I ran it locally and it seems like it chugged along just fine.

Reply:
Works for me!